commonmark-ext-yaml-front-matter/src/main/java/org/commonmark/ext/yaml/YAMLFrontMatterBlock.java

```diff
@@ -0,0 +1,6 @@
+package org.commonmark.ext.yaml;
+
+import org.commonmark.node.CustomBlock;
+
+public class YAMLFrontMatterBlock extends CustomBlock {
```

Comment:
Can you change the `YAML` in all class names to `Yaml`? This is to make it consistent with `Html` in `HtmlRenderer`.